src/styles/utilities/_flex.scss

```diff
@@ -1,4 +1,34 @@
-.u-flex-row {
-    display: flex;
-    flex-direction: row;
-}
\ No newline at end of file
+@use "../abstracts" as *;
+
+.u-flex {
+	display: flex;
+}
+
+/**
+ * Direction
+ */
+
+$direction: (
+	"col": column,
+	"row": row,
+);
+@include generate-utils("flex", "flex-direction", $direction, ("md"));
+
+/**
+ * Align
+ */
+
+$align: (
+	"center": center,
+);
+@include generate-utils("align", "align-items", $align);
+
+/**
+ * Justify
+ */
+
+$justify: (
+	"center": center,
+	"between": space-between,
+);
+@include generate-utils("justify", "justify-content", $justify, ("md"));
```

Comment:
Hoe maken we de afweging tussen context-class (`.p-home__element`) & util-class? Tot nu toe hebben we heel traditioneel gewerkt maar met deze util klasses komen we dan eerder in een [Tailwind](https://tailwindcss.com/)-achtige opzet. Lijkt me niet slecht om de redenering hierrond even te documenteren.

Reply:
Dat is een beetje de afweging maken van ga ik veel styling nodig hebben voor een element of niet.
Bij kleinere blokken met weinig styling (titels, marges tussen elementen) -> utils
Bij grotere blokken met veel styling (responsive, layout, components ...) -> custom element
Daar kunnen guidelines/docs voor voorzien worden.